megaqc/user/forms.py

```diff
@@ -58,6 +58,8 @@ def validate(self):
 
 class RegisterForm(Form):
     """Register form."""
+    class Meta:
+        csrf = False # disable csrf protection on registration
```

Comment:
Why do we want to disable this for the registration page?

Reply:
No idea. It was disabled for that page in the config, but was throwing warnings about that method 
of disabling CSRF being deprecated. Can re-enable if there wasn't a particular reason for it.